@here/harp-vectortile-datasource/lib/OmvDataFilter.ts

```diff
@@ -713,7 +713,7 @@ export class ComposedDataFilter implements OmvFeatureFilter {
      */
     get hasKindFilter() {
         return this.filters.reduce<boolean>(
-            (result, filter) => result && filter.hasKindFilter,
```

Comment:
should be a separate commit, it seems unrelated to the rest of the PR

Reply:
I had to add a lot of missing unit tests for VectorTileDecoder, VectorTileDataProcessor, OmvDataAdapter... I found this bug while unit testing VectorTileDataProcessor, that's why I fixed it here.
It could be moved to a separate PR, I'd have to move the failing test as well. But since it's a one liner, is it ok if I leave it here?

Reply:
All good, ok, sounds fine.